<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/Министерство_природных_ресурсов_и_экологии_Российской_Федерации__Минприроды_России_-removebg-preview.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="path_line">
          <property name="styleSheet">
           <string notr="true">border-radius: 2px;
padding:5px;
border: 1px solid #DCDCDC;
min-height: 25px;
margin-top: 20px
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_path">
          <property name="styleSheet">
           <string notr="true">background-color: #D3D3D3;
padding: 7px;
border-radius: 4px;
min-width: 120px;
min-height: 25px;
margin-top: 20px</string>
          </property>
          <property name="text">
           <string>Выбрать источник</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="info">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #696969;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QCheckBox" name="b_grafic">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Получить график распределения</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="strart_b">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>154</width>
          <height>69</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>69</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2E8B57;
padding: 7px;
border-radius: 4px;
min-width: 140px;
min-height: 25px;
margin-bottom: 30px;
color: white;
font-size: 16px;
</string>
        </property>
        <property name="text">
         <string>Начать</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #DCDCDC;
padding: 0 5px 0 5px;
min-height: 45px;
max-height::65px;
height: 40px;
min-width: 140px;
max-width::300px;
font-size: 15px;
</string>
          </property>
          <property name="text">
           <string>Всего фотографий</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLCDNumber" name="All">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">min-height: 45px;
max-height::65px;
height: 40px;
min-width: 140px;
max-width::300px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLCDNumber" name="Kos">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLCDNumber" name="Kab">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="l_Kos">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #DCDCDC;
padding: 0 0 0 5px;</string>
          </property>
          <property name="text">
           <string>Косуля</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLCDNumber" name="Ole">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="l_Kab">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #DCDCDC;
padding: 0 5px 0 5px;</string>
          </property>
          <property name="text">
           <string>Кабарга</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="l_Ole">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #DCDCDC;
padding: 0 0 0 5px;</string>
          </property>
          <property name="text">
           <string>Олень</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
